docs/Common/Enums/index.md

```diff
@@ -936,8 +937,10 @@
 |SIXTH|11||
```

Comment:
please, add description (per https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0266-New-vehicle-data-GearStatus.md#update-enum-prndl-1)
to 	
```
<element name="DRIVE">
+		<description>Regular Drive mode</description>
```

Reply:
fixed in a9b04f5839d95c8aaf7e0a88e14cb1d1cb0d1643